odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -4566,6 +4566,12 @@ def test_compute_inverse(self):
         self.assertEqual(record.foo, 'Bar')
         self.assertEqual(record.bar, 'Bar')
 
+    def test_x2many_inverse(self):
+        record = self.env['test_new_api.compute.inverse'].create(
+            {'one2many_f': [Command.create({'foo': 'test'})]}
+        )
+        self.assertTrue(all(child.id for child in record.one2many_f), record.one2many_f)
```

Comment:
The message should be a string, it is confusing to read.
```suggestion
        self.assertTrue(all(child.id for child in record.one2many_f), f"Should be database records: {record.one2many_f!r}")
```